src/main/resources/api/openapi.yaml

```diff
@@ -1030,3 +1076,140 @@ paths:
             text/plain; charset=UTF-8:
               schema:
                 type: string
+
+  /wallet/balances:
+    get:
+      summary: Get total amount of confirmed Ergo tokens and assets
+      operationId: walletBalances
+      tags:
+        - wallet
+      responses:
+        '200':
+          description: Get total amount of confirmed Ergo tokens and assets
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BalancesSnapshot'
+        default:
+          description: Error
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+
+
+  /wallet/balances/unconfirmed:
+    get:
+      summary: Get amount of unconfirmed Ergo tokens and assets
+      operationId: walletBalancesUnconfirmed
+      tags:
+        - wallet
+      responses:
+        '200':
+          description: Get amount of unconfirmed Ergo tokens and assets
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BalancesSnapshot'
+        default:
+          description: Error
+          content:
+            text/plain; charset=UTF-8:
+              schema:
+                type: string
+
+  /wallet/transaction:
```

Comment:
We already have this method in transactions endpoint, I don't think we need to similar methods, at least one should be removed